tools/cabana/binaryview.cc

```diff
@@ -62,19 +62,27 @@ void BinaryView::addShortcuts() {
   QShortcut *shortcut_endian = new QShortcut(QKeySequence(Qt::Key_E), this);
   QObject::connect(shortcut_endian, &QShortcut::activated, [=]{
     if (hovered_sig != nullptr) {
+      const Signal *hovered_sig_prev = hovered_sig;
       Signal s = *hovered_sig;
       s.is_little_endian = !s.is_little_endian;
       emit editSignal(hovered_sig, s);
+
+      hovered_sig = nullptr;
+      highlight(hovered_sig_prev);
```

Comment:
can call highlight(hovered_sig) here, hovered_sig will not changed after refresh, no need to save it to a temp variable hovered_sig_prev.